src/main/kotlin/net/atos/zac/zoeken/converter/ZaakZoekObjectConverter.kt

```diff
@@ -55,7 +55,7 @@ class ZaakZoekObjectConverter @Inject constructor(
             // we use the name of this enum in the search index
             vertrouwelijkheidaanduiding = zaak.vertrouwelijkheidaanduiding.name
             isAfgehandeld = !zaak.isOpen
-            zgwApiService.findInitiatorRoleForZaak(zaak).ifPresent { setInitiator(it) }
+            zgwApiService.findInitiatorRoleForZaak(zaak)?.let(::setInitiator)
```

Comment:
Setting another value from within a `let` doesn't feel right to me, it implies that a value is returned, but we are not. Using `run` implies that an arbitrary block of code is executed in the value's context...

Reply:
true. switched to using `also`